examples/lua/README.md

```diff
@@ -0,0 +1,64 @@
+
+
+```bash
+nats-sub "Bucket_Notification"
+```
+
+
+[Full documentation for subscribing](https://docs.nats.io/nats-server/clients).
+
+Alternatively, configure the script to point to an existing NATS broker by editing the following part in the script to match the parameters of your existing nats server. 
+
+```
+local nats_params = {
+        host ='{host}',
+        port = {port},
+}
+```
+
+## Usage
+
+* Upload the [script]():
+
+```bash
+radosgw-admin script put --infile=script.lua --context=postRequest
+```
+* Add the packages used in the script:
+
+```bash
+radosgw-admin script-package add --package=nats --allow-compilation
+radosgw-admin script-package add --package=lunajson --allow-compilation
+radosgw-admin script-package add --package='lua-cjson 2.1.0-1' --allow-compilation
+```
+* Restart radosgw.
+* create a bucket:
+```
+s3cmd --host=localhost:8000 --host-bucket="localhost:8000/%(bucket)" mb s3://mybucket
+```
+* upload a file to the bucket and make sure that the nats server received the notification
+
+```
+s3cmd --host=localhost:8000 --host-bucket="localhost:8000/%(bucket)" put hello.txt s3://mybucket
+```
+
+Expected output:
+```
+Received on [Bucket_Notification]: '{"Records":{"eventId":"","s3":{"s3SchemaVersion":"1.0","configurationId":"mynotif1","bucket":{"ownerIdentity":{"principalId":"testid"},"id":"1e9f15f3-ee3c-4d0b-8a7c-16606865701f.4137.1","arn":"arn:aws:s3:default::mybucket","name":"mybucket"},"object":{"tags":[{"x-amz-content-sha256":"1fd78297d07615091ece92f56b7be6384f784a25603fb0209d0d77ba088ecd6e","x-amz-date":"20210628T121434Z","x-amz-meta-s3cmd-attrs":"atime:1624882471/ctime:1624882467/gid:1247/gname:matan/md5:f387a7f32576586c8890f3cf54bd4c42/mode:33204/mtime:1624882467/uid:1247/uname:matan","x-amz-storage-class":"STANDARD"}],"size":3303,"key":"hello.txt","versionId":"","sequencer":"F7E6D75DC742D108","metadata":[{"x-amz-content-sha256":"1fd78297d07615091ece92f56b7be6384f784a25603fb0209d0d77ba088ecd6e","x-amz-date":"20210628T121434Z","x-amz-meta-s3cmd-attrs":"atime:1624882471/ctime:1624882467/gid:1247/gname:matan/md5:f387a7f32576586c8890f3cf54bd4c42/mode:33204/mtime:1624882467/uid:1247/uname:user","x-amz-storage-class":"STANDARD"}],"eTag":"37b51d194a7513e45b56f6524f2d51f2"}},"eventName":"ObjectCreated:Put","responseElements":{"x-amz-request-id":"1e9f15f3-ee3c-4d0b-8a7c-16606865701f.74107.3","x-amz-id-2":"1217b-default-default"},"eventSource":"ceph:s3","eventVersion":"2.1","awsRegion":"default","opaqueData":"me@example.com","requestParameters":{"sourceIPAddress":""},"userIdentity":{"principalId":"testid"},"eventTime":"2021-06-28 12:14:34"}}'
+```
+
+## Requirements
+* Lua 5.3
+* Luarocks
+* Lua-devel
+
+Full Installation instructions for [Lua(5.3) and luarocks](https://github.com/luarocks/luarocks/wiki/Installation-instructions-for-Unix). 
```

Comment:
don't think this is needed.
lua/luarocks can be installed from official repos for ubuntu and fedora, and from epel on centos